app/helpers/blacklight/url_helper_behavior_decorator.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# OVERRIDE blacklight 6 to allow full url for show links for shared search and override blacklight url tracking
+module Blacklight
```

Comment:
Until we figure out UTK's decorator issue, we'll prob have to convert this to module_eval once pulled into UTK. @laritakr

Reply:
Most likely. I will have to test it to make sure when I pull it in.